ReportPortal.SpecFlowPlugin/ReportPortalHooks.cs

```diff
@@ -196,6 +196,8 @@ public static void AfterFeature(FeatureContext featureContext)
                         ReportPortalAddin.OnAfterFeatureFinished(null, new TestItemFinishedEventArgs(Bridge.Service, request, currentFeature, featureContext, null));
                     }
                 }
+
+                ReportPortalAddin.RemoveFeatureTestReporter(featureContext, currentFeature);
```

Comment:
FeatureTestReporter should be removed only by the last thread otherwise other threads will create multiple test items for a single feature or will not be able to create scenarios. The line should be moved inside the loop above it.